src/generator/data/base_url

```diff
@@ -0,0 +1 @@
+https://www.unicode.org/Public/12.1.0
```

Comment:
Missing terminating newline...

Reply:
Also, does this need to be updated to update to a new version of Unicode? Your documentation didn't say.

Reply:
> Missing terminating newline...

This is intentional, the content of this file is read as base URL in the `dune` file thus requires no trailing newline. Explained in the `README.md` now!

Reply:
In the README you say "leading" rather than "trailing" newline. You may want to fix that.